src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
```

Comment:
The message 'Excavator started its work.' is not very informative. It would be better to indicate that the Excavator is currently working, for example: 'Excavator is working.'